INSTALL.md

```diff
@@ -1,8 +1,22 @@
 # Install Open Source Event Manager
-All the information that you need to install OSEM. If you have any problems with installing don't hesitate to [contact us](https://github.com/openSUSE/osem#contact)
+This file contains all the information you need to install OSEM. If you run into any problems, do not hesitate to [contact us](https://github.com/openSUSE/osem#contact).
+
+If you encounter
+```
+An error occurred while installing yajl-ruby (1.2.0), and Bundler cannot continue.
+Make sure that `gem install yajl-ruby -v '1.2.0'` succeeds before bundling.
+```
+or
+
+```
+An error occurred while installing json (1.8.3), and Bundler cannot continue.
+Make sure that `gem install json -v '1.8.3'` succeeds before bundling.
+```
+
+check that your Ruby version is not greater than `ruby-2.3.3`.
```

Comment:
We require Ruby 2.1.2, right @hennevogel @ChrisBr ?
But still I don't think we should list all possible/known issues in our installation guide. Perhaps the README and/or INSTALLATION file(s) should be mentioning the ruby and the rails versions.